ui/src/app/views/workspace/component.html

```diff
@@ -1,103 +1,16 @@
 <div class="workspace-container">
   <div class="main-body">
-    <h2>{{workspace.name}}</h2>
-    <div style="display: flex; justify-content: space-between">
-      <div style="margin-left: -0.5rem;">
-        <button class="btn btn-link" [class.selected]="tabOpen === Tabs.Notebooks" (click)="tabOpen = Tabs.Notebooks">
-          Notebooks ({{notebookList.length}})
-        </button>
-        <button class="btn btn-link final" [class.selected]="tabOpen === Tabs.Cohorts" (click)="tabOpen = Tabs.Cohorts">
-          Cohorts ({{cohortList.length}})
-        </button>
+    <h2 style="height: 65px">{{greeting}}</h2>
+    <div class="card-area">
```

Comment:
Until My PR is in, this is going to block access to review cohorts. I don't think we want to do that yet? (There's sorta some weird dependencies here...)